test/src/main/configurations/MainConfig/ConfigurationReceivers.xml

```diff
@@ -10,7 +10,7 @@
 						timestampField="TTIMESTAMP"
 						commentField="TVARCHAR"
 						maxCommentLength="100"
-						selectCondition="TNUMBER=${jdbc.receivertest.key}0"
+						selectCondition="TNUMBER=${jdbc.receivertest.key}0 AND TVARCHAR2='${hostname}'"
```

Comment:
You might be able to avoid this new column by setting 
`jdbc.receivertest.key=1${sun.java.launcher.pid}`
similar to:
`jdbc.querylistener.key=${sun.java.launcher.pid}`
(add the prefix to avoid collision with other tests)